### PR TITLE
Fix duplicate LocalVariableTypeTable check

### PR DESCRIPTION
**Pull request by theresa-m**
fixes: https://github.com/eclipse/openj9/issues/8890

Signed-off-by: Theresa Mammarella <Theresa.T.Mammarella@ibm.com>

### CODE REVIEW

**Comment by DanHeidinga** [comment]
Is this ready to be moved out of draft?

File: runtime/bcutil/ClassFileOracle.cpp [hunk 1]
```diff
@@ -1271,6 +1271,8 @@ ClassFileOracle::walkMethodCodeAttributeAttributes(U_16 methodIndex)
 							J9CfrLocalVariableTypeTableEntry *lvttCompareEntry = &(localVariableTypeTable->localVariableTypeTable[localVariableTypeTableCompareIndex]);
 							if ((lvttEntry->startPC == lvttCompareEntry->startPC)
 								&& (lvttEntry->length == lvttCompareEntry->length)
+								&& (lvttEntry->nameIndex == lvttCompareEntry->nameIndex)
+								&& (lvttEntry->signatureIndex == lvttCompareEntry->signatureIndex)
```
Comment (DanHeidinga) [inline runtime/bcutil/ClassFileOracle.cpp:1275]: This is a good starting point.  We should also, possibly under a separate PR, validate whether this should be checking the contents for equality, not the indexes

Comment (DanHeidinga) [reply runtime/bcutil/ClassFileOracle.cpp:1275]: Would be worth testing the RI as well

Comment (theresa-m) [reply runtime/bcutil/ClassFileOracle.cpp:1275]: Thats a good idea. There appears to be at least one more missing LocalVariableTypeTable check that will come after this in a separate pr.

**Comment by pshipton** [comment]
jenkins test sanity zlinux jdk14

**Comment by theresa-m** [comment]
created an issue for myself to look into string equality: https://github.com/eclipse/openj9/issues/9405

**Pull request merged**